docs/install.md

```diff
@@ -4,13 +4,14 @@ Installing Chunky
 Before installing Chunky you will need Java 8 update 40 or later.
 [You can download Java here.](http://java.com)
 
-On Ubuntu, and some other Linux distributions, you may have to install a JavaFX
+On Ubuntu, some other Linux distributions, and with Java 11 or greater you may have to install a JavaFX
 library to be able to run Chunky. On Ubuntu 16.04, just have to install the
-package `openjfx`.
+package `OpenJFX`.
```

Comment:
The package really is called `openjfx`, as in `apt-get install openjfx`

Reply:
Thanks for that correction. I'll also add the command as an example just in case.